master/internal/workspace/authz_iface.go

```diff
@@ -44,6 +44,7 @@ type WorkspaceAuthZ interface {
 	CanCreateWorkspace(ctx context.Context, curUser model.User) error
 	CanCreateWorkspaceWithAgentUserGroup(ctx context.Context, curUser model.User) error
 	CanCreateWorkspaceWithCheckpointStorageConfig(ctx context.Context, curUser model.User) error
+	CanAddWorkspaceNamespaceBindings(ctx context.Context, curUser model.User, workspaceID int32) error
```

Comment:
Same comment about changing it to Modify

Reply:
Ref [above reply](https://github.com/determined-ai/determined/pull/9226#discussion_r1622588286)